app/assets/javascripts/components/locales/he.jsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Note for Contributors:
+ * This file (he.jsx) is based on en.jsx and should be updated once in a while with new releases.
+ * To make other contributors' life easier, please REMEMBER:
+ *   1. to add your new string here; and
+ *   2. to remove old strings that are no longer needed; and
+ *   3. to sort the strings by the key.
+ *   4. Respect the language standards we have set here please!
+ * Thanks!
+ */
```

Comment:
it helped me when I compared to the English string file to keep the line count so I can more easily detect changes that need updates to the translation file. Is that a problem?